examples/api/lib/material/reorderable_list/reorderable_list_view.1.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for ReorderableListView
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: _title,
+      home: Scaffold(
+        appBar: AppBar(title: const Text(_title)),
+        body: const MyStatefulWidget(),
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  final List<int> _items = List<int>.generate(50, (int index) => index);
+
+  Widget _proxyDecorator(Widget child, int index, Animation<double> animation) {
+    return AnimatedBuilder(
+      animation: animation,
+      builder: (BuildContext context, Widget? child) {
+        final double animValue = Curves.easeInOut.transform(animation.value);
+        final double elevation = lerpDouble(0, 6, animValue)!;
+        return Material(
+          elevation: elevation,
+          color: Colors.blue,
```

Comment:
It would be nice if color and colorScheme were colorScheme.primary and colorScheme.primaryVariant. We want to encourage developers to use the color scheme.

If this function was defined within the build method, you could refer to two additional local color variables that were based on the colorScheme (like odd,evenItemColor).

Reply:
Interesting, I notice the default `colorScheme.secondary` is exactly the color I wanted to showcase in the demo, darker blue. Do you think it is good for this?

https://user-images.githubusercontent.com/48603081/140181359-f785a80a-4b33-47f9-9fa2-762562b5b557.mp4

Reply:
This looks great!